.github/workflows/deploy.yml

```diff
@@ -1,10 +1,11 @@
-name: GitHub Pages
+name: Build Stats
 
 on:
   push:
     branches:
       - main
   pull_request:
+  workflow_dispatch:
```

Comment:
Why do we need it?

Reply:
For testing workflow in branches other than `main`.